json_schema/type/protocol/biomaterial_collection/dissociation_protocol.json

```
@@ -54,7 +54,56 @@
                 "type": "object",
                 "$ref": "module/process/purchased_reagents.json"
             },
-            "user_friendly": "Protocol reagents"
+            "user_friendly": "Purchased reagents"
+        },
+        "digestion_time": {
+            "description": "Time of digestion of the specimen.",
+            "type": "integer",
+            "example": "2; 5; 10",
+            "user_friendly": "Digestion time"
+        },
+        "digestion_time_unit": {
+            "description": "The unit in which the digestion time is expressed.",
+            "type": "object",
+            "$ref": "module/ontology/time_unit_ontology.json",
+            "user_friendly": "Digestion time unit"
+        },
+        "digestion_temperature": {
+            "description": "Temperature of digestion in Celsius or in general terms (i.e. cold, warm, room temperature).",
+            "pattern": "\\d+(\\.\\d+)?|frozen|cold|room temperature|warm",
```

Pattern must be anchored (And it should be grouped in order to properly capture alternative groups):
```suggestion
            "pattern": "^(\\d+(\\.\\d+)?|frozen|cold|room temperature|warm)$",
```

---

Otherwise, things like "11frozen", "frozencold" will be accepted https://regex101.com/r/iRzCpr/1